.github/workflows/ci.yml

```diff
@@ -1,10 +1,33 @@
 ---
-name: Pulp CI
-on:
-  push:
-    branches:
-      - '*'
+name: Pulp PR
+on: pull_request
 jobs:
+  git-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.1
+      - run: |
+          git fetch --prune --unshallow
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - name: Install requests
+        run: pip install requests
+      - name: Check commit message
+        env:
+          GITHUB_CONTEXT: ${{ github.event.pull_request.commits_url }}
+        run: |
+          for sha in $(curl $GITHUB_CONTEXT | jq '.[].sha' | sed 's/"//g')
+          do
+            python .github/validate_commit_message.py $sha
+            VALUE=$?
+            if [ "$VALUE" -gt 0 ]; then
+              exit $VALUE
+            fi
+          done
+        shell: bash
+      - name: Check submodule status
+        run: |
+          .ci/submodule_check.sh
```

Comment:
It's really hard to review this block of code because it is showing up as new code rather than a renamed pull_request.yml.

Suggestions on a different git diff command to run or something?

You might be able to do the delete in 1 commit, and then the rename/changes in the other.

(This assumes that we should be renaming the .yml file where the pull_request is.)

Reply:
Here's the sequence of events:

* Deleted the FIPS job from release + PR CI
* Deleted ci.yml
* Renamed pull_request.yml to ci.yml to be more consistent with other repos